Gemfile

```diff
@@ -27,6 +27,7 @@ group :development, :test do
   gem "pry-rails"
   gem "ffaker"
   gem "rails-controller-testing"
+  gem "stripe"
```

Comment:
Why do we need to introduce this development dependency? Is it just for testing purposes?

Reply:
@kennyadsl Exactly, I used the Stripe gem in the specs as a source for "ground-truth" customer and payment info.